README.md

```diff
@@ -598,8 +598,59 @@ The buildpack includes a variety of telemetry agents that can be configured to c
 
 ### New Relic
 
+#### Set up New Relic integration
+
+[Fluent Bit](https://docs.fluentbit.io/manual/) is used to collect Mendix Runtime logs to [New Relic](https://newrelic.com/).
+
+The metrics are collected by the [New Relic Java Agent](https://docs.newrelic.com/docs/apm/agents/java-agent/getting-started/introduction-new-relic-java/) and an integration with the [Telegraf agent](https://docs.influxdata.com/telegraf/).
+The first one collects container and database metrics, while the second collects metrics related to the Mendix Runtime.
```

Comment:
Can we point to the NR docs where it says what the NR java agent collects ?

Reply:
Done